src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -118,6 +118,10 @@ Released 2023-Oct-26
   definition.
   ([#4990](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4990))
 
+* `dns.lookups.duration` metric is renamed to `dns.lookup.duration`. This change
```

Comment:
interesting...there is no code change here. Do we plan to keep updating changelog in otel instrumentation, for every change in .net repo? Or this is a one-time thing and we dont expect anything futher?

Reply:
I mentioned this one because we have a changelog entry in previous release with different name. In future, I don't think we need to mention every change. Except when the change might impact users moving from lower version to net8 or above.

Reply:
Going forward, I think we should simply update the README to point to the Microsoft official docs which lists these metrics and let users track the changes from there.